src/components/CallView/shared/LocalVideoControlButton.vue

```diff
@@ -124,6 +135,9 @@ export default {
 
 	mounted() {
 		subscribe('local-video-control-button:toggle-video', this.updateDeviceState)
+		if (this.startWithoutMediaEnabled) {
+			this.model.disableVideo()
```

Comment:
Let's do it in CallView `setup`?
We have localMediaModel there, and it always mounts when joining a call. Buttons can be reused elsewhere, logic doesn't really belong to them

Reply:
Sounds like a good approach